test/catalog/constraints_test.cpp

```diff
@@ -43,8 +65,6 @@ namespace test {
 class ConstraintsTests : public PelotonTest {};
 
 // FIXME: see the explanation rpc_client_test and rpc_server_test
-TEST_F(ConstraintsTests, BlankTest) {}
-/*
 #ifdef NOTNULL_TEST
```

Comment:
It's mostly for debugging purpose. Sometimes we just want to run a few tests to check for memory / pointer issues.